docs/docs/manual/sortview.md

```diff
@@ -0,0 +1,29 @@
+---
+id: sortview
+title: Sort and view
+sidebar_label: Sort and view
+---
+
+## Sort
+
+You can temporarily sort your rows by one column. You can sort:
+*   text alphabetically or reverse
+*   numbers by largest or smallest
+*   dates by earliest or latest
+*   boolean values by false, then true, or by **core-views/true-false**
+
+You can also choose where to place errors and blank cells in the sorting. Text can be case-sensitive or not: cells that start with lowercase characters will appear ahead of uppercase.
+
+![A screenshot of the Sort window.](/img/sort.png)
+
+After you apply sorting, you can make it permanent, remove it, reverse it, or apply a different **sorting**. You’ll find “Sort” in the project grid header to the right of the rows-display setting, which will show all current sorting settings. 
```

Comment:
I think an explanation of in what sense sorts are usually not "permanent" (i.e. the original row/record order is preserved which can be seen by the numbers in the All column) and what making it "permanent" means (rows/records are renumbered sequentially based on the current sort and the "sort" disappears

Also think we need an explanation of how multiple sorts apply - i.e. if you apply multiple sorts at once, they are ordered and don't override each other (sort by author then title preserves the author orders and sorts titles within a single author).

This links to making sorts "permanent" as to get a sort of books by author then by title within author you'd want to:

Apply an author sort
Apply a title sort
Make the sort permanent